docs/malware-scanner/README.md

```diff
@@ -1 +1,44 @@
-## *Currently empty, check back later or contribute yourself*
\ No newline at end of file
+# Malware scanner and detector
+
+---
+
+# Dynamic analysis 
+A slow but reliable detector looking for jar capabilities, potential malware sign
+- run static analysis to filter known threats
+- downloading stuff from the internet (potentially jars, but can be genuine)
+- zip/jar manipulation.
+- able to execute stuff (Runtime.exec())
```

Comment:
Phrasing: We should make it _VERY_ clear the scanner does not execute stuff, but detects the ability to do so.